Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ministryofjustice/ruby:2-webapp-onbuild
+
+RUN touch /etc/inittab
+
+ENV UNICORN_PORT 3000
+EXPOSE $UNICORN_PORT
+
+COPY run.sh /run.sh
```

Comment:
Is it standard behaviour to put `run.sh` in `/`? It feels more natural to me to have it in a location equivalent to its location on the host (e.g. `/usr/src/app/run.sh`)?